composer.json

```diff
@@ -23,6 +23,7 @@
         "symfony/validator": "~2.3||~3.0.0||~3.1.0||^3.2.4||~4.0",
         "symfony/filesystem": "~2.3||~3.0||~4.0",
         "symfony/config": "~2.3||~3.0||~4.0",
+        "symfony/translation": "~2.3||~3.0||~4.0",
```

Comment:
We've accomplished this in the latest build so it can be removed from the changeset after this work is rebased.